pkg/hubble/observer/option/option.go

```diff
@@ -12,12 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package serveroption
+package option
```

Comment:
why this change?

Reply:
I was on requested by @Rolinh to prepare for the peer API. I'll drop the commit for now.